www/docs/admin/policies.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+include_once '../../includes/easyparliament/init.php';
+#include_once INCLUDESPATH . 'easyparliament/commentreportlist.php';
+#include_once INCLUDESPATH . 'easyparliament/searchengine.php';
+include_once INCLUDESPATH . 'easyparliament/member.php';
+#include_once INCLUDESPATH . 'easyparliament/people.php';
+
+$this_page = 'admin_policies';
+
+$db = new ParlDB;
+
+$scriptpath = '../../../scripts';
+
+$PAGE->page_start();
+$PAGE->stripe_start();
+
+$out = '';
+if (get_http_var('editpolicy') && get_http_var('action') === 'Save') {
+    $out = update_policy();
+}
+
+if (get_http_var('editpolicy')) {
+    $out .= edit_policy_form();
+} else {
+    $out .= list_policies();
+}
+
+$subnav = subnav();
+
+print '<div id="adminbody">';
+print $subnav;
+print $out;
+print '</div>';
+
+function edit_policy_form() {
+    global $db;
+    $policyid = get_http_var('editpolicy');
+    $query = "SELECT policy_id, title, description, image, image_attrib, image_license, image_license_url, image_source FROM policies WHERE policy_id = :policy_id";
+    $q = $db->query($query, array(
+        ':policy_id' => $policyid
+    ));
+
+    $out = '';
+
+    for ($row = 0; $row < $q->rows(); $row++) {
+
+        $out = "<h3>Edit policy " . $q->field($row, 'title') . "</h3>\n";
+
+        $out .= '<form action="policies.php?editpolicy=' . $q->field($row, 'policy_id') . '" method="post">';
+        $out .= '<input name="action" type="hidden" value="Save">';
+        $out .= '<label for="title">Title</label>';
+        $out .= '<span class="formw"><input id="title" name="title" type="text"  size="60" value="' . $q->field($row, 'title') . '"></span>' . "<br>\n";
```

Comment:
None of these are being HTML escaped, I don't know if title/description could have " in them...